src/components/map/map-control.js

```diff
@@ -261,26 +261,30 @@ const MapDrawPanel = React.memo(({
           <ToolbarItem
             onClick={() => onSetEditorMode(EDITOR_MODES.EDIT)}
             label="select"
-            icon={(<CursorClick height="22px"/>)}
+            iconHeight="22px"
```

Comment:
are you using this value in toolbarItem?

Reply:
unless we need the height, i like the previous version where we just pass the icon. More concise